LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT LICENSE
+
+Copyright (c) 2017 Andrew Edwards
```

Comment:
Since both Andrew and I work on this library, would it make more sense to have it as `Copyright (c) 2017 Anthony Castelli & Andrew Edwards` I haven't dealt with licenses for open source projects before.

Reply:
I am not sure if the name in the license has any meaning, but I just changed it as you suggested.